docs/sources/clients/promtail/stages/json.md

```diff
@@ -22,6 +22,10 @@ json:
 
   # Name from extracted data to parse. If empty, uses the log message.
   [source: <string>]
+
+  # If true then any lines that cannot be successfully parsed as valid JSON objects
+  # will be dropped instead of being sent to Loki.
+  drop_malformed: <bool>
```

Comment:
Is there a default value?   I think that the syntax here might be incomplete.  I think that there should be "[]" surrounding the configuration variable if this variable is optional.

If there is a default value and this configuration variable is optional, then the definition ought to be revised to something like

```suggestion
  # When true, any lines that cannot be successfully parsed as valid JSON objects
  # will be dropped instead of being sent to Loki (the default).
  [drop_malformed: <bool> | default = false]
```